src/main/java/com/linkedin/xinfra/monitor/services/ConsumeService.java

```diff
@@ -40,7 +40,7 @@
 import org.apache.kafka.common.metrics.Metrics;
 import org.apache.kafka.common.metrics.MetricsReporter;
 import org.apache.kafka.common.metrics.Sensor;
-import org.apache.kafka.common.metrics.stats.Total;
+import org.apache.kafka.common.metrics.stats.CumulativeSum;
```

Comment:
Do you know if the changes to the metrics up stream will change the computation of any of the metrics produced by the monitor?

Reply:
Addressed in the above PR

Reply:
I don't have a specific change I'm asking for.  I'm asking if you understand the changes that were done to Kafka that resulted in Total becoming CumulativeSum.  Are there any other side effects as part of that change to open source Kafka?

Reply:
See KIP-488 --
https://cwiki.apache.org/confluence/display/KAFKA/KIP-488%3A+Clean+up+Sum%2CCount%2CTotal+Metrics 

In previous cases, no consistency existed in naming convention with a missing metric, the quadrant for which was redundantly covered. With the new changes in Kafka, the metrics including the CumulativeSum for "Sum" are vividly named with all quadrants covered in a unique way -- no ambiguity +structure of titles show a clear pattern that maps to correct metrics.

`Non-sampled Sum Metric`:
- (deprecated) org.apache.kafka.common.metrics.stats.Total
- (new) org.apache.kafka.common.metrics.stats.CumulativeSum

Does that answer your question?

![image](https://user-images.githubusercontent.com/8682887/95449211-4b793880-0919-11eb-9aff-dd1421b39627.png)